feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -191,3 +192,28 @@ export enum CartValidationStatusCode {
   PRICING_ERROR = 'pricingError',
   UNRESOLVABLE_ISSUES = 'unresolvableIssues',
 }
+
+export interface AddToCartToastConfig {
+  timeout?: number;
+}
```

Comment:
I think the convention is to declare the config types in the feature-config.ts file, like cart-config.ts.
If this is the case, declare `AddToCartToastConfig` in cart-config.ts